### ISSUE TITLE
 question from limit involving a float

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#2017]
Imported from SourceForge on 2024-07-05 16:43:59
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2011-08-05 23:59:58
Original: https://sourceforge.net/p/maxima/bugs/2249

---

The limit code has far too much  code that fills the fact data base with  comparisons to 1.0\*10^8 or
1.0\*10^-8. One of these silly facts leaks out:

\(%i16\) integrate\( t\*exp\(-\(t-a\)^2\), t\)$

Silly question:

\(%i17\) limit\(%,t,minf\);
"Is  "t+a+1.0\*10^8"  positive or negative?"neg;
\(%o17\) 0

\(%i18\) build\_info\(\);
Maxima version: 5.25.0
Maxima build date: 12:0 8/2/2011
Host type: i686-pc-mingw32
Lisp implementation type: Clozure Common Lisp
Lisp implementation version: Version 1.7-r14925M  \(WindowsX8632\)
\(%o18\)

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 16:44:00
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2023-11-19 12:00:49
Original: https://sourceforge.net/p/maxima/bugs/2249/#ee5b

---

- **status**: open --> closed
- **Group**:  --> None

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 16:44:04
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2023-11-19 12:00:49
Original: https://sourceforge.net/p/maxima/bugs/2249/#5aa1

---

This bug has been fixed--it doesn't happen with Maxima 5.47.  Maybe this should be added to the testsuite, but an asksign it hard to detect.  I'm going to close this bug

~~~
(%i18)	integrate( t*exp(-(t-a)^2), t);
(%o18)	(%i*((%i*gamma_incomplete(1,(2*a-2*t)^2/4)*(2*a-2*t)^2)/(2*t-2*a)^2-(%i*gamma_incomplete(1/2,(2*a-2*t)^2/4)*a*(2*a-2*t))/abs(2*t-2*a)))/2
(%i19)	factor(ratsimp(diff(%,t)));
(%o19)	t*%e^(-t^2+2*a*t-a^2)
(%i20)	ratsimp(%);
(%o20)	t*%e^(-t^2+2*a*t-a^2)
(%i21)	build_info();
(%o21)	build_info(version="5.47.0",timestamp="2023-06-02 20:31:42",host="x86_64-w64-mingw32",lisp_name="SBCL",lisp_version="2.3.2",

~~~
